corehq/apps/app_manager/views/modules.py

```diff
@@ -198,6 +198,7 @@ def _get_shared_module_view_context(request, app, module, case_property_builder,
         'case_list_form_options': _case_list_form_options(app, module, lang),
         'valid_parents_for_child_module': _get_valid_parents_for_child_module(app, module),
         'shadow_parent': _get_shadow_parent(app, module),
+        'case_types': get_case_types_for_domain(app.domain),
```

Comment:
Can you filter out the module's own case type?

Reply:
This is actually intentional. In the UI, the text for this is `Case List and Case Search Types`. This would leave the flexibility of whether to include the module's case-type in the case-list/search lists. Also it would handle the case where the user may change the module's case-type.